animals_and_tips.html

```diff
@@ -142,7 +140,7 @@ <h2>Hover on each image to discover information about them</h2>
 <div class="container">
 	<img src="images/Zebra.jpg" alt="Zebra" class="image">	
 	<div class="overlay overlayTop">
-		<div class="animals_text"><h1>Zebra</h1><h2>Equus quagga</h2><p>They are far more aggressive and a lot more dangerous. Zebras have been known to kick each other to death, they will viciously bite any human that comes too close, and there are even many accounts of zebras killing lions, so make sure you stay very far away.
+		<div class="animals_text"><h1>Zebra</h1><h2>Equus quagga</h2><p>Zebras are far more aggressive and a lot more dangerous than you think. Zebras have been known to kick each other to death, they will viciously bite any human that comes too close, and there are even many accounts of zebras killing lions, so make sure you stay very far away.

```

Comment:
I made sure to check if the information was legible and easy to understand and correct some simple errors to clarify the information, even though it was grammatically correct